src/plugins/intel_gna/tests/functional/shared_tests_instances/subgraph_tests/input_conv_concat.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_tests/input_conv_concat.hpp"
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+
+using namespace SubgraphTestsDefinitions;
+
+namespace {
+
+using namespace std;
+using namespace InferenceEngine;
+
+const vector<Precision> net_precisions = {Precision::FP32, Precision::FP16};
+
+const vector<map<string, string>> configs = {{{"GNA_DEVICE_MODE", "GNA_SW_FP32"}},
```

Comment:
Is the intention here to run it only on default target?

Reply:
Added a few more targets.